queue_job/models/queue_job.py

```diff
@@ -506,3 +508,25 @@ def _test_job(self, failure_rate=0):
         _logger.info("Running test job.")
         if random.random() <= failure_rate:
             raise JobError("Job failed")
+
+    def _call_api(self, **kwargs):
+        webhook_url = (
+            self.env["ir.config_parameter"].sudo().get_param("queue_job.webhook_url")
+        )
+        if webhook_url:
+            data = kwargs.get("data", None)
+            json = kwargs.get("json", None)
+            headers = kwargs.get("headers", None)
+            try:
+                res = requests.post(
+                    url=webhook_url, data=data, json=json, headers=headers, timeout=20
```

Comment:
`timeout=20` is too much, better if you take inspiration from the fire and forget strategy of odoo, see https://github.com/odoo/odoo/blob/18.0/odoo/addons/base/models/ir_actions.py#L868